File: backend/modules/sabor-familia/src/main/java/br/com/puc/saborfamilia/service/mensagem/dto/response/RemoverMensagemResponse.java
package br.com.puc.saborfamilia.service.mensagem.dto.response;

import java.time.LocalDateTime;

public record RemoverMensagemResponse(
  Long conversaId,
  MensagemResponse mensagem,
  String ultimaMensagem,
  LocalDateTime dataUltimaMensagem,
  Long naoLidas
) {
}

File: frontend/src/features/sabor_familia/dto/menssagem/response/RemoverMensagemResponse.ts
import type { MensagemResponse } from "./MensagemResponse";

export interface RemoverMensagemResponse {
  conversaId: number;
  mensagem: MensagemResponse;
  ultimaMensagem: string | null;
  dataUltimaMensagem: string | null;
  naoLidas: number;
}
